tooling/car/merge.go

```diff
@@ -4,8 +4,8 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/ipfs/boxo/ipld/car/v2/blockstore"
 	"github.com/ipfs/go-cid"
-	"github.com/ipld/go-car/v2/blockstore"
```

Comment:
@galargh that's the fix for the crazy bug I had before,
- I double-checked: `init()` is called only once per package as we expected. 
- When I renamed `fixture.go` into `zfixture.go`, the bug was fixed 🎉
- I realized this had to be a collision between new boxo packages and the old ones.

Reply:
🤯